mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/positionTooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+interface TooltipProps {
+  /**
+  * Tooltip's children (ReactNode)
+  */
+  children: ReactNode;
+  /**
+  * Tooltip's content (string | ReactNode)
+  */
+  content: string | ReactNode;
+  /**
+  * Tooltip's class-style (string)
+  */
+  className: string;
+  /**
+  * Tooltip's position relative cursor, optional (default - PositionTooltip.TOP)
```

Comment:
You have ts-doc features like {@link something} etc. Use it and will not have to read and syncronize all project docs after some variable renamings)

Reply:
PositionTooltip.TOP is not a text it is a part of code. Link it

Reply:
I got it. Done